sycl/cmake/modules/AddSYCL.cmake

```diff
@@ -10,7 +10,7 @@ function(add_sycl_library LIB_NAME TYPE)
   target_link_libraries(${LIB_NAME} PRIVATE ${ARG_LIBRARIES})
 
   if (ARG_TOOLCHAIN)
```

Comment:
Probably we need to remove the option then.

Reply:
Doesn't look like we were using it anyway, so it has been removed!